code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -28,7 +28,7 @@
 /obj/item/gun/energy/kinetic_accelerator/examine(mob/user)
 	. = ..()
 	if(max_mod_capacity)
-		. += "<b>[get_remaining_mod_capacity()]%</b> mod capacity remaining."
+		. += "<b>[get_remaining_mod_capacity()]%</b> mod capacity remaining. You can use a crowbar to remove modules"
```

Comment:
Needs `.` at end

Reply:
follow the formatting.
new line, then
`<span class='info'>You can use a <b>crowbar </b> to remove modules.</span>`